<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Processing_Form</class>
 <widget class="QWidget" name="Processing_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_preprocessing">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>预处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_feature_extraction">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>特征提取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_model_training">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模型训练</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_processing_back_navigator">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回导航界面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_processing_help">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_data_analysis">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据分析</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
